### ISSUE TITLE
com.google.code.regexp.Pattern class isn't Serializable (java.util.regex.Pattern is)

### ISSUE THREAD

**Issue by juanplopes** [tony19/named-regexp#3]

**Comment by av-ast** [comment]
Hello,

I'm trying to serialize compiled pattern...
```java
SerializationUtils.serialize(com.google.code.regexp.Pattern.compile("pattern"));
```
... but it fails with
```
incompatible types: com.google.code.regexp.Pattern cannot be converted to java.io.Serializable
```

Could you please help me?

**Comment by tony19** [comment]
@av-ast The fix is actually in 0.2.4 not yet released. I'll release it today/tomorrow.

In the meantime, you can load the SNAPSHOT by adding this to your build config:

```
repositories {
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
}

dependencies {
    compile 'com.github.tony19:named-regexp:0.2.4-SNAPSHOT'
}
```

**Comment by tony19** [comment]
Released in 0.2.4

**Comment by av-ast** [comment]
@tony19 thanks for your help and such a quick reply!
